sdk/core/azure-core/azure/core/polling/base_polling.py

```diff
@@ -447,6 +447,30 @@ def initialize(self, client, initial_response, deserialization_callback):
         except OperationFailed as err:
             raise HttpResponseError(response=initial_response.http_response, error=err)
 
+    def get_continuation_token(self):
+        # type() -> str
+        import pickle
+        return pickle.dumps(self._initial_response)
```

Comment:
Why is the continuation token the initial response and not the latest received response?

Reply:
good question, because I have no guarantee that the internediate response (like a GEt from an azure-asyncoperation) will give me enough information to actually restart. For instance, in case I have both azure-asyncoperation and Location, the final GET will be on Location, and I won't find it back in the intermediate calls.
The initial request always gives me enough information to restart, in the scenario where it's already done it means two calls:
- Get on azure-asyncoperation: status is Success
- Get on resource/Location for final resource

Reply:
Storing the initial response is simple, but wide for sure. I could make this smaller but would need to create my own serialization format, and enable a PollingMethod to restart from that format, which I can skip if I just serialize the initial response as-is

Reply:
Thanks! Sounds good. This is on the base polling method correct?
In which case, if one were to define a custom polling method they could set the continuation token to which ever value/response would be allow them to restart?

Reply:
absolutely, the poller delegate continuation_token mgmt to the PollingMethod entirely